packages/react-components/react-spinner/src/components/Spinner/renderSpinner.tsx

```diff
@@ -15,8 +15,11 @@ export const renderSpinner_unstable = (state: SpinnerState) => {
       {state.label && shouldRenderSpinner && (labelPosition === 'above' || labelPosition === 'before') && (
         <state.label />
       )}
-
-      {state.spinner && shouldRenderSpinner && <state.spinner />}
+      {state.spinner && shouldRenderSpinner && (
```

Comment:
I don't think we need the `state.spinner` check anymore?

Reply:
The spinner prop can be set to `null` to prevent it from being rendered, so it has to be checked here still.